mips/pmap.c

```diff
@@ -347,12 +354,13 @@ void tlb_exception_handler(exc_frame_t *frame) {
   vm_map_t *map = get_active_vm_map_by_addr(vaddr);
   if (!map) {
     klog("No virtual address space defined for %08lx!", vaddr);
-  } else {
-    vm_prot_t access = (code == EXC_TLBL) ? VM_PROT_READ : VM_PROT_WRITE;
```

Comment:
Why has this change been reverted? The semantics are preserved and the code is more readable.
Should this change go in a separate PR?

Reply:
If you want to clean up the code please move such changes to separate PR.

I understand there's a temptation to clean up the code as we go, but that distracts the reviewer. Thinking of separation of concerns applies to preparing patches as well.